app/javascript/dashboard/store/modules/conversations/helpers.js

```diff
@@ -32,6 +33,10 @@ export const applyPageFilters = (conversation, filters) => {
     const filterByLabels = labels.every(label => chatLabels.includes(label));
     shouldFilter = shouldFilter && filterByLabels;
   }
+  if (conversationType === 'unattended') {
+    const hasNoFirstReply = !firstReplyOn;
+    shouldFilter = shouldFilter && hasNoFirstReply;
```

Comment:
```suggestion
    shouldFilter = shouldFilter && !firstReplyOn;
```

The variable `hasNoFirstReply` is not at all useful. Please consider the comment.